Gems/Multiplayer/Code/Include/Multiplayer/MultiplayerConstants.h

```diff
@@ -9,11 +9,12 @@
 #pragma once
 
 #include <AzCore/EBus/Event.h>
+#include <AzCore/Metrics/IEventLoggerFactory.h>
```

Comment:
This file can forward declared `enum class AZ::Metrics::EventLoggerId : AZ::u32;` and avoid the need to include the IEventLogger.h

The IEventLogerFactory.h isn't actually needed either as it is IEventLogger.h that provides the EventLoggerId enum

Reply:
Done